server/dumpbagserver/templates/dumps.html

```diff
@@ -1,44 +1,43 @@
 {% extends "layout.html" %}
 {% set active_page = "dumps" %}
 {% block body %}
-
-<div class="dumps">
```

Comment:
is this removal on purpose? If yes, there is an unmatched `</div>` at the end of the file

Reply:
![Screenshot from 2020-07-13 16-20-12](https://user-images.githubusercontent.com/29477079/87315752-167d6580-c525-11ea-9e46-5ba6d57db22b.png)

Reply:
This was in the style so it was an oversight on my part, i added it back. Thank you.